src/struct_ls/red_black_constantcoef_gs.c

```diff
@@ -316,7 +316,7 @@ hypre_RedBlackConstantCoefGS( void               *relax_vdata,
                                 x_dbox, stencil_shape[offd[5]]);
                      xoff4 = hypre_BoxOffsetDistance(
                                 x_dbox, stencil_shape[offd[4]]);
-                  /* fall through */
+                     HYPRE_FALLTHROUGH;
```

Comment:
What is this?  And why?

Reply:
This informs the compiler that it's safe to fall through the next case statement. By default, modern compilers assume that's not safe, thus producing a compilation warning. So with this macro, we don't get the warning